### PR TITLE
FUSETOOLS-1784 - Put focus on First Tab Item

### PR DESCRIPTION
**Pull request by apupier**
- for destination: "Basic"
- for server: "Mandatory"

### CODE REVIEW

File: jboss-fuse-sap-tool-suite/plugins/org.fusesource.ide.sap.ui/src/org/fusesource/ide/sap/ui/properties/uicreator/IDataUICreator.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.ui.views.properties.tabbed.TabbedPropertySheetWidgetFactory;
+
+/**
+ * @author Aurelien Pupier
+ *
+ */
+public interface IDataUICreator {
+
+	void createControls(Composite basicContainer, TabbedPropertySheetWidgetFactory widgetFactory);
```
Comment (lhein) [inline jboss-fuse-sap-tool-suite/plugins/org.fusesource.ide.sap.ui/src/org/fusesource/ide/sap/ui/properties/uicreator/IDataUICreator.java:22]: we should write api doc for public methods

File: jboss-fuse-sap-tool-suite/plugins/org.fusesource.ide.sap.ui/src/org/fusesource/ide/sap/ui/properties/uicreator/IDestinationDataUICreator.java [hunk 2]
```diff
@@ -12,17 +12,13 @@
...
 /**
  * @author Aurelien Pupier
  *
  */
-public interface IDestinationDataUICreator {
-
-	void createControls(Composite basicContainer, TabbedPropertySheetWidgetFactory widgetFactory);
+public interface IDestinationDataUICreator extends IDataUICreator {
 
 	void initDataBindings(DataBindingContext bindingContext, EditingDomain editingDomain, DestinationDataStoreEntryImpl destinationDataStoreEntry);
```
Comment (lhein) [inline jboss-fuse-sap-tool-suite/plugins/org.fusesource.ide.sap.ui/src/org/fusesource/ide/sap/ui/properties/uicreator/IDestinationDataUICreator.java:23]: API Doc

File: jboss-fuse-sap-tool-suite/plugins/org.fusesource.ide.sap.ui/src/org/fusesource/ide/sap/ui/properties/uicreator/IServerDataUICreator.java [hunk 3]
```diff
@@ -12,17 +12,13 @@
...
 /**
  * @author Aurelien Pupier
  *
  */
-public interface IServerDataUICreator {
-
-	void createControls(Composite container, TabbedPropertySheetWidgetFactory widgetFactory);
+public interface IServerDataUICreator extends IDataUICreator {
 
 	void initDataBindings(DataBindingContext bindingContext, EditingDomain editingDomain, ServerDataStoreEntryImpl serverDataStoreEntry);
```
Comment (lhein) [inline jboss-fuse-sap-tool-suite/plugins/org.fusesource.ide.sap.ui/src/org/fusesource/ide/sap/ui/properties/uicreator/IServerDataUICreator.java:23]: API Doc

**Comment by lhein** [comment]
+1

File: jboss-fuse-sap-tool-suite/plugins/org.fusesource.ide.sap.ui/src/org/fusesource/ide/sap/ui/properties/uicreator/IDestinationDataUICreator.java (hunk 2 above)
Comment (apupier) [inline jboss-fuse-sap-tool-suite/plugins/org.fusesource.ide.sap.ui/src/org/fusesource/ide/sap/ui/properties/uicreator/IDestinationDataUICreator.java:23]: Nothing is part of API in this plugin. Exported packages are exported only to x-friends which are the tests plugins

**Pull request merged**